llvm/lib/CodeGen/SelectionDAG/DAGCombiner.cpp

```diff
@@ -2872,6 +2906,10 @@ SDValue DAGCombiner::visitADD(SDNode *N) {
   // Try to match AVGFLOORU fixedwidth pattern
   if (SDValue V = combineFixedwidthToAVGFLOORU(N, DAG))
     return V;
+  
+  // Try to match AVGFLOORU fixedwidth pattern
+  if (SDValue V = combineFixedwidthToAVGFLOORS(N, DAG))
```

Comment:
Merge this into combineFixedwidthToAVGFLOORU (rename to combineFixedwidthToAVGFLOOR)